### PR TITLE
InputArea remove hasCounter class

### PR DESCRIPTION
**Pull request by aarnoldaas**
- Removed has-counter class that has weird margin
- It breaks grid layout
- Fixed tests. It just tested that class is present that class had only margin-bottom :/

This is breaking change I think

### CODE REVIEW

**Comment by aarnoldaas** [comment]
Any news?

File: src/InputArea/InputArea.driver.js [hunk 1]
```diff
@@ -20,7 +21,8 @@ const inputAreaDriverFactory = ({element, wrapper, component}) => {
     getTabIndex: () => textArea.tabIndex,
     getReadOnly: () => textArea.readOnly,
     getResizable: () => textAreaElement.classList.contains(styles.resizable),
-    getHasCounter: () => textAreaElement.classList.contains(styles.hasCounter),
+    getHasCounter: () => !!$component.find(counterSelector).length,
+    getCounterValue: () => $component.find(counterSelector).text(),
```
Comment (shlomitc) [inline src/InputArea/InputArea.driver.js]: can you please add this to the `README.TESTKIT.md` file?

**Comment by aarnoldaas** [comment]
I do not understand what examples should I add.
I think it's a bug. How inputArea can have margin bottom 30px?
This is a breaking change because probably everybody applied hack around this weird behavior. 
If I'm wrong and inputArea should have margin, just close pull request.

**Comment by argshook** [comment]
i think this change is fine, i find only handful of projects that use it.
ill rebase this branch on master, add automated storybook for `<InputArea/>` and merge

**Pull request merged**